operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -235,7 +235,25 @@ local utils = import 'mixin-utils/utils.libsonnet';
       OK: $._colors.success,
       success: $._colors.success,
       'error': $._colors.failed,
-      cancel: '#A9A9A9',
+      Canceled: '#A9A9A9',
```

Comment:
don't we have both Canceled and cancel?
<img width="1494" alt="Screenshot 2025-03-17 at 16 19 39" src="https://github.com/user-attachments/assets/12fe0b4c-6715-42cf-8d92-7150f48859e5" />

Reply:
@dimitarvdimitrov I can't find where that status code actually comes from because everything with it in ops appears to be aggregated. It's not on any of our dashboard metrics that I can find, but I will add `cancel` back in.

Reply:
i think it's coming from dskit https://github.com/search?scopeName=All+repos&scope=&q=org%3Agrafana+%22%5C%22cancel%5C%22%22+repo%3Agrafana%2Fdskit+&type=code